R/import-gr-skey-tif-to-pg-rast.R

```diff
@@ -80,6 +107,11 @@ import_gr_skey_tif_to_pg_rast <- function(
                   password = pg_conn_param["password"][[1]],
                   port = pg_conn_param["port"][[1]])
   RPostgres::dbExecute(conn, statement = qry1)
+
+  if (tolower(geom_type) == 'polygon' | tolower(geom_type) == 'multipolygon' ) {
+    qry2 <- qry3
+  }
+  else if (tolower(geom_type) == 'centroid'){print('Centroid geometry selected')}else(print('Geometry type not recognized, default to centroids'))
```

Comment:
Is there an ending curly brace in this if / else if / else section? I don't see one but it could be the PR screen..  is one needed?

Also, to keep consistent formatting (tabbing / spacing etc):

```
else if (tolower(geom_type) == 'centroid') {
    print('Centroid geometry selected')
} else {
    print('Geometry type not recognized, default to centroids')
}
```